src/openrct2/world/Map.h

```diff
@@ -151,7 +151,10 @@ SmallSceneryElement* map_get_small_scenery_element_at(int32_t x, int32_t y, int3
 EntranceElement* map_get_park_entrance_element_at(int32_t x, int32_t y, int32_t z, bool ghost);
 EntranceElement* map_get_ride_entrance_element_at(int32_t x, int32_t y, int32_t z, bool ghost);
 EntranceElement* map_get_ride_exit_element_at(int32_t x, int32_t y, int32_t z, bool ghost);
-int32_t tile_element_height(int32_t x, int32_t y);
+int16_t tile_element_height(int32_t x, int32_t y);
+uint8_t map_get_highest_land_height(int32_t xMin, int32_t xMax, int32_t yMin, int32_t yMax);
+uint8_t map_get_lowest_land_height(int32_t xMin, int32_t xMax, int32_t yMin, int32_t yMax);
+int16_t tile_element_water_height(int32_t x, int32_t y);
```

Comment:
This should only be the one new function and the change in return value

Reply:
Did you relocate the other three functions? A bit confused on this

Reply:
I removed them but the build still fails. Any reason for this?

Reply:
The build error is quite clear:
```
../src/openrct2-ui/../openrct2/actions/SmallSceneryRemoveAction.hpp:135:27: error: 'tile_element_height' was not declared in this scope
         res->Position.z = tile_element_height(res->Position.x, res->Position.y);
                           ^~~~~~~~~~~~~~~~~~~
```

Reply:
@janisozaur So, tile_element_height still must remain? I know one of my other members pushed an update to fix all of these conflicts, but it doesn't seem like that fixed the appveyor build nor the traviscl.